### ISSUE TITLE
Add more instruments to corresponding role

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#463]
Low priority. More instruments should be added as a criteria for http://id.loc.gov/vocabulary/relators/itr to assign people to the right roles. Listing missing instruments:
* clarinet (http://lobid.org/hbz01/HT019318164)
* guitar (http://lobid.org/hbz01/HT017716661)
* piano (http://lobid.org/hbz01/HT019318164)
* Violoncello (http://lobid.org/hbz01/HT018102322)

**Comment by ChristophEwertowski** [comment]
Definitions for existing roles expanded (number of affected resources in parentheses):
[Instrumentalist](http://id.loc.gov/vocabulary/relators/itr): Violine (5854), Violoncello (3731), Cembalo (2480), Flöte (1909), Klarinette (1154), Gitarre/guitar (1133/184), Schlagzeug/drums (1121/146), Oboe (1101), Trompete (1066), Horn (614), Posaune (532), Fagott (481), Saxophon (464), Laute (454), Kontrabass (417)
[Contributor](http://id.loc.gov/vocabulary/relators/ctb): Mitarb. (85795), Beitr. (4228)
[Singer](http://id.loc.gov/vocabulary/relators/sng): Mezzosporan (4099), Counterten (918)
[Editor](http://id.loc.gov/vocabulary/relators/edt): editor (3015), Ed. (1071), Herausgeber (1022)
[Honoree](http://id.loc.gov/vocabulary/relators/hnr): Widmungsempfänger (2115)
[Creator](http://id.loc.gov/vocabulary/relators/cre): Textverf. (84801), VerfasserIn (1200) (in regular expression as "Verfasser"), author (780)
[Libretto](http://id.loc.gov/vocabulary/relators/lbt): lbt (1273)
[Artist](http://id.loc.gov/vocabulary/relators/art): Künstler (530)
[Conductor](http://id.loc.gov/vocabulary/relators/cnd): Chorleit. (456)

Newly added roles:
[Dancer](http://id.loc.gov/vocabulary/relators/dnc): Tänzer (3765)
[Choreograph](http://id.loc.gov/vocabulary/relators/chr): Choerogr. (3639)
[Moderator](http://id.loc.gov/vocabulary/relators/mod): Moderation (179)

And then there are some roles which I'm not sure how to sort in (will go on with it tomorrow):
- Bearb. (229319) -> ctb?
- aut (author; 57227) -> cre?
- Begr. (49052) -> org
- Text (29771)
- oth (other; 16305) -> ctb?
- dgg (degree granting institution; 15096)
- Komm. (10413)
- orm (organizer; 10155)
- arr (arranger; 5512)
- Vorl. (5327)
- Bühnenbild (4967)
- Vorr. (4525)
- Vorwort (4351) / wpr (writer of preface; 4475) / Vorw. (2409)
- wst (writer of supplementary texual content; 4210)
- com (compiler; 3988)
- Adresat (3923)
- dgs (degree supervisor; 3481)
- Kamera (3476)
- win (writer of introduction; 3040)
- nrt (narrator; 2336) -> spk?
- Choreinstud. (2325)
- wat (writer of added text; 1954)
- lyr (lyricist; 1729) -> cre?
- wac (writer of added commentary; 1637)
- fmd (film director; 1534)
- Kostüm (1503)
- cmm (commentator; 1493)
- Komment. (1259)
- his (host institution; 1256)

**Comment by acka47** [comment]
+1 for the first batch with exception of  "Textverf." where I am not sure. (See comment on "Text" below.)

Re. the unclear cases, we generally should always use the three-letter code where it is provided (I also added the relator URIs to those in the list below):
- Bearb. (229319) -> ctb?* A: +1 for `ctb` but that's our default anyway*
- aut (author; 57227) -> cre? *A: There actually is an "author" relator code see http://id.loc.gov/vocabulary/relators/aut. Nonetheless, I think `cre` probably is the better solution.*
- Begr. (49052) -> org (orginator) :question: 
- Text (29771) -> cre (example: http://lobid.org/hbz01/HT017608824) *A: I don't think we should touch this. In case of the linked resource it is an opera where the text is not the most important part.*
- oth (other; 16305) -> ctb? *A: There is a code `oth` but I think we should stick with ctb.*
- dgg (degree granting institution; 15096) -> ctb *A: Why not usehttp://id.loc.gov/vocabulary/relators/dgg?*
- Komm. (10413) -> probably "Kommentator", ctb *A: Is it "commentator" in the meaning of http://id.loc.gov/vocabulary/relators/dgg? Probably better stick to ctb.*
- orm (organizer; 10155) -> ctb *A: http://id.loc.gov/vocabulary/relators/orm*
- arr (arranger; 5512) -> ctb or cmp (composer) *A: http://id.loc.gov/vocabulary/relators/arr*
- Vorl. (5327) (Vorlage) -> org A: :question:
- Bühnenbild (4967) -> cre A: :question: 
- Vorr. (4525) -> spk (speaker), if it does mean "Vorredner" *A: I think http://id.loc.gov/vocabulary/relators/wpr would be the right choise*
- Vorwort (4351) / wpr (writer of preface; 4475) / Vorw. (2409) -> new role wpr or ctb? *A: http://id.loc.gov/vocabulary/relators/wpr plus the Vorr. from above*
- wst (writer of supplementary texual content; 4210) -> ctb *A: +1 for ctb*
- com (compiler; 3988) -> cre *A: http://id.loc.gov/vocabulary/relators/com*
- Adressat (3923) -> new role rcp (Adressee) *A: +1 for rcp*
- dgs (degree supervisor; 3481) -> ctb *A: http://id.loc.gov/vocabulary/relators/dgs*
- Kamera (3476) -> new role vdg (videographer) *A: not sure about this*
- win (writer of introduction; 3040) -> ctb? *A: http://id.loc.gov/vocabulary/relators/win*
- nrt (narrator; 2336) -> *A: http://id.loc.gov/vocabulary/relators/nrt*
- Choreinstud. (2325) -> mus (musician) *A: don't know*
- wat (writer of added text; 1954) -> ctb *A: http://id.loc.gov/vocabulary/relators/wat*
- lyr (lyricist; 1729) -> cre? *A: http://id.loc.gov/vocabulary/relators/lyr*
- wac (writer of added commentary; 1637) -> ctb *A: http://id.loc.gov/vocabulary/relators/wac*
- fmd (film director; 1534) -> drt (director) *A: http://id.loc.gov/vocabulary/relators/fmd*
- Kostüm (1503) -> new role cst *A: +1 for cst*
- cmm (commentator; 1493) / Komment. (1259) -> new role cmm *A: +1 for cmm*
- his (host institution; 1256) -> ctb *A: http://id.loc.gov/vocabulary/relators/his*

**Comment by ChristophEwertowski** [comment]
Like you proposed I added the right URLs to the MARC relator codes (except for "oth" and "wst" which are mapped to `contributor`). For "Bühnenbild" and "Choreinstudierung" I added new lobid vocabulary roles. "Vorr." isn't used in the branch because I don't really know what it means (and a cataloging collegue couldn't help me neither). "Vorl." is already mapped to http://id.loc.gov/vocabulary/relators/ant. The string "Kamera" isn't used anymore because of it's ambiguity of taking pictures or filming something. 
Because I didn't add any examples the most important task is to check if the German labels fit.
